src/packages/src/xroad/redhat/SOURCES/opmonitor/xroad-opmonitor-initdb.sh

```diff
@@ -73,6 +81,7 @@ then
 
   db_url=$(get_prop ${db_properties} op-monitor.hibernate.connection.url)
   db_user=$(get_prop ${db_properties} op-monitor.hibernate.connection.username "$db_user")
+  db_conn_user="${db_user}${suffix}"
```

Comment:
duplicates the suffix if the db_user is already in format user@suffix (as written on line 151)